R/loadModule.R

```diff
@@ -67,7 +67,9 @@ loadModule <- function( module, what = character(), loadNow,
         loadNow <- !is(loadM, "error")
     }
     if(loadNow) {
-        .botched <- isBotchedSession()
+        ## .botched <- isBotchedSession()
+        .botched <- FALSE
```

Comment:
That is why I brought this to the `rcpp-core` list and John -- AFAIK he added it.